packages/safety-suite-api/src/api/permission/grants.ts

```diff
@@ -5,17 +5,35 @@ import {ApiOptions, ApiResponse, CustomerSpecificQuery, Id} from '../../types';
 import {DeleteResponse, GrantDTO, InputGrant} from './types';
 
 /**
- * Gets a specific `Grant`
- * @param userId ID of the `User` related to the grant.
+ * Gets a list of all user grants grouped by customer alias.
+ * Requires super admin grant.
+ * @param userId ID of the `User` related to the grants.
+ * @param apiOptions Optional options for runApi.
+ * @returns Array of `Grant` objects.
+ */
+export const getGrantsGroupedByCustomer = (
+  userId: Id,
+  apiOptions?: ApiOptions
+): Request<ApiResponse<Record<string, GrantDTO[]>>> =>
+  runApi({
+    method: 'GET',
+    urlRoot: 'permissionUrlRoot',
+    route: `/api/grants/${userId}`,
+    apiOptions
+  });
+
+/**
+ * Gets all user grants for a specific customer.
+ * @param userId ID of the `User` related to the grants.
  * @param query Object containing query params for this route.
  * @param apiOptions Optional options for runApi.
  * @returns A single `Grant` object.
```

Comment:
The return in the jsdocs needs updated since it's returning an array, but it says single.